gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -389,6 +394,19 @@ private void addHDFSFiles(String jobFileList) throws IOException {
     }
   }
 
+  private void addHdfsJars(String hdfsJarFileList) throws IOException {
```

Comment:
Why is this needed when `JOB_JAR_FILES_KEY` can be used to specify jar files? The only difference I see is that `JOB_JAR_FILES_KEY` specifies jars on the local filesystem.